.github/workflows/allo-allo.yml

```
@@ -40,10 +44,121 @@ jobs:
+              env:
+                  AUTHOR: "${{ github.event.pull_request.user.login }}"
+                  REPO: "${{ github.event.repository.name }}"
+                  OWNER: "${{ github.event.repository.owner.login }}"
+              run: |
+                  echo "AUTHOR=${AUTHOR}" >> $GITHUB_ENV
+                  echo "REPO=${REPO}" >> $GITHUB_ENV
+                  echo "OWNER=${OWNER}" >> $GITHUB_ENV
+
+            - name: "Count Merged Pull Requests"
+              id: "count_merged_pull_requests"
+              uses: "actions/github-script@v6"
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)